src/ClientHandle.cpp

```diff
@@ -1884,6 +1884,19 @@ void cClientHandle::HandleUseItem(bool a_UsedMainHand)
 
 
 
+// @param a_Status 0: successfully loaded, 1: declined, 2: failed download, 3: accepted
+void cClientHandle::HandleResourcePack(UInt8 a_Status)
+{
+	if (cRoot::Get()->GetPluginManager()->CallHookResourcePack(a_Status))
```

Comment:
depending https://github.com/cuberite/cuberite/pull/5440/files#r979201537
this stay like that or is and'd with the config file response